kuksa_databroker/Dockerfile-cli

```diff
@@ -0,0 +1,47 @@
+# /********************************************************************************
+# * Copyright (c) 2022 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+# This is expected to be executed in the kuksa.val top-level directory
+# This buuilds the databroker-cli
```

Comment:
Minor spelling error "buuilds"

Reply:
Certainly true, and fixed now.

Would still be good to hear whether the container created by this Docker file actually  also _works_ :)